>>> lazy_matrix_mul([[1, 2], [3, 4]], [[1, 2], [3, 4]])
[[ 7 10]
 [15 22]]
>>> lazy_matrix_mul('holberton', [[1, 2], [3, 4]])
Traceback (most recent call last):
...
numpy.core._exceptions.UFuncTypeError: ufunc 'multiply' did not contain a loop with signature matching types (dtype('<U21'), dtype('<U21')) -> dtype('<U21')
>>> lazy_matrix_mul([[1, 2], [3, 4]], 'holberton')
Traceback (most recent call last):
...
numpy.core._exceptions.UFuncTypeError: ufunc 'multiply' did not contain a loop with signature matching types (dtype('<U21'), dtype('<U21')) -> dtype('<U21')
>>> lazy_matrix_mul([[]],[[1, 2], [3, 4]] )
Traceback (most recent call last):
...
ValueError: shapes (1,0) and (2,2) not aligned: 0 (dim 1) != 2 (dim 0)
>>> lazy_matrix_mul([[1, 2], [3, 4]], [[]] )
Traceback (most recent call last):
...
ValueError: shapes (2,2) and (1,0) not aligned: 2 (dim 1) != 1 (dim 0)
>>> lazy_matrix_mul([[1, 2], ['3', 4]],[[1, 2], [3, 4]])
Traceback (most recent call last):
...
ValueError: data type must provide an itemsize
>>> lazy_matrix_mul([[1, 2], [3, 4]],[['1', 2], [3, 4]])
Traceback (most recent call last):
...
ValueError: data type must provide an itemsize
>>> lazy_matrix_mul([[1, 2, 6], [3, 4]],[[1, 2], [3, 4]])
Traceback (most recent call last):
...
[list([1, 2, 6, 3, 4, 3, 4, 3, 4])
 list([1, 2, 6, 1, 2, 6, 3, 4, 3, 4, 3, 4, 3, 4])]
>>> lazy_matrix_mul([[2, 6], [3, 4]],[[1, 1, 2], [3, 4]])
Traceback (most recent call last):
...
[list([1, 1, 2, 1, 1, 2, 3, 4, 3, 4, 3, 4, 3, 4, 3, 4, 3, 4])
 list([1, 1, 2, 1, 1, 2, 1, 1, 2, 3, 4, 3, 4, 3, 4, 3, 4])]
>>> lazy_matrix_mul([[1,2, 6], [1,3, 4]],[[1, 2], [3, 4]])
Traceback (most recent call last):
...
ValueError: shapes (2,3) and (2,2) not aligned: 3 (dim 1) != 2 (dim 0)
